collectors/apps.plugin/apps_plugin.c

```diff
@@ -2533,6 +2574,11 @@ static inline int collect_data_for_pid(pid_t pid, void *ptr) {
 
 static int collect_data_for_all_processes(void) {
     struct pid_stat *p = NULL;
+    
+    // clear process state counter
+    for (proc_state i = RUNNING; i < END; i++) {
```

Comment:
Another suggestion here,  use `memset(proc_states, 0, sizeof(proc_state));` instead the loop.

Reply:
thanks for this tip